rosidl_typesupport_connext_cpp/bin/rosidl_typesupport_connext_cpp

```diff
@@ -3,6 +3,8 @@
 import argparse
 import sys
 
+from rosidl_cmake import extract_message_types
+from rosidl_parser import UnknownMessageType
```

Comment:
This one needs rosidl_parser as a dependency. It will probably not fail due to transitive, but should be there.